src/modules/users/components/UserProfile/UserMeta.tsx

```diff
@@ -33,11 +45,16 @@ const UserMeta = ({
       />
     </div>
     <div className={styles.headingContainer}>
-      <Heading
-        appearance={{ margin: 'none', size: 'medium', theme: 'dark' }}
-        text={displayName}
-        data-test="userProfileName"
-      />
+      {displayName && (
+        <Heading
+          appearance={{ margin: 'none', size: 'medium', theme: 'dark' }}
+          text={displayName}
+          data-test="userProfileName"
+        />
+      )}
+      <Link className={styles.profileLink} to="/edit-profile">
```

Comment:
On my machine the icon is not inlined with the username, although I see in your demo that it is. Can you please double check ?

![Screenshot from 2019-10-04 17-07-31](https://user-images.githubusercontent.com/1193222/66214028-92f73400-e6c9-11e9-93d5-83c7da8e9bef.png)

Reply:
Hmmmm I can't seem to replicate that... 🤔

Cleared cache, tried another machine.... which browser are you using?